Add a path from original zod error to final messsages (#295)
igorkamyshev/farfetched: .changeset/dull-foxes-grow.md

---
'@farfetched/zod': patch
---

> Improve zodContract error messages with path from original zod error

Reviewer: ```suggestion
Add a path from original zod error to final messsages
```
